R/getRunStatus.R

```diff
@@ -142,14 +142,12 @@ getRunStatus <- function(mydir = dir(), sort = "nf", user = NULL) {
           iters <- suppressWarnings(system(paste0("grep -A 30 'PARAMETER p80_repy  sum' ", fulllst), intern = TRUE))
           if (length(iters) > 0) {
             iters <- grep("^$|--|modelstat", iters, invert = TRUE, value = TRUE)
```

Comment:
calculate number of regions and provide convergence information for all of them (relevant for H21)